tool/travis.sh

```diff
@@ -15,7 +15,7 @@ dartanalyzer --fatal-warnings \
   test/all.dart
 
 # Run the tests.
-dart -c test/all.dart
+dart --conditional_directives -c test/all.dart
```

Comment:
Actually, I think we can remove the DDC section below; the strong-mode lie in the `.analysis_options` should give us coverage of any strong mode issues.